### ISSUE TITLE
Odd documentation generation with upgrade to version 0.19.0

### ISSUE THREAD

**Issue by PatrickRice-KSC** [hashicorp/terraform-plugin-docs#360]
### Terraform CLI and terraform-plugin-docs Versions

Terraform version: 1.4.0
Plugin docs version: v0.19.0

### Provider Code

```go
https://gitlab.com/gitlab-org/terraform-provider-gitlab/-/merge_requests/1888/diffs#9d055e3689fb5193976798648b837563c2baeeb0
```

### Expected Behavior

- Headers for nested data source schemas includes the name of the attribute
- All description content is included in the front matter as expected

### Actual Behavior

- Headers for nested data source schemas seem to be removing the name of the attribute
- One link seems to be removed from the "description" front matter of the generated files

### Steps to Reproduce

1. `tfplugindocs generate`

### How much impact is this issue causing?

Low

### Additional Information

Is this potentially related to https://github.com/hashicorp/terraform-plugin-docs/commit/ab779d3b106c5c7e804f38068549540afce3bea1 ? It seems like the attribute schemas were being _properly_ generated in v0.18.0 though, I don't think an empty header is a great user experience here :)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SBGoods** [comment]
Hi @PatrickRice-KSC 👋🏾,
I'm sorry that you're running into trouble here. The issue that you are having with your headers looks similar to a bug reported in #356 where the headers were not rendering properly for nested object attributes. A fix #357 has been merged in main and will be released in `v0.19.1` early next week.

The issue where URL has been removed from the description in your docs is a bug related to the `plainmarkdown` function which I've opened a PR for the fix in #361. This fix will also be included in the next `v0.19.1` patch release.
